src/Amazon.Lambda.Tools/LambdaDefinedCommandOptions.cs

```diff
@@ -417,5 +417,29 @@ public static class LambdaDefinedCommandOptions
                 ValueType = CommandOption.CommandOptionValueType.StringValue,
                 Description = "Docker image name and tag in the 'name:tag' format."
             };
+        public static readonly CommandOption ARGUMENT_EPHEMERAL_STORAGE_SIZE =
+            new CommandOption
+            {
+                Name = "Ephemerals Storage Size",
+                Switch = "--ephemerals-storage-size",
+                ValueType = CommandOption.CommandOptionValueType.IntValue,
+                Description = "The size of the function's /tmp directory in MB."
+            };
+        public static readonly CommandOption ARGUMENT_FUNCTION_URL_ENABLE =
+            new CommandOption
+            {
+                Name = "Function Url Enable",
+                Switch = "--function-url-enable",
+                ValueType = CommandOption.CommandOptionValueType.BoolValue,
+                Description = "Enable function URL."
+            };
+        public static readonly CommandOption ARGUMENT_FUNCTION_URL_AUTH =
+            new CommandOption
+            {
+                Name = "Function Url Auth Type",
+                Switch = "--function-url-auth",
+                ValueType = CommandOption.CommandOptionValueType.StringValue,
+                Description = "The type of authentication that your function URL uses, default value is NONE. Valid values: NONE or AWS_IAM"
```

Comment:
Can the valid values be data driven through (for example) an SDK enum? This would keep the help text from getting stale if a new enum was added

Reply:
I switched the code to use the constant values from the SDK but we don't have a mechanism to easily iterate through the available values. If new values are added to the list it is likely we would have to make code changes to handle the function resource policy.